src/api/neonDB.js

```diff
@@ -63,18 +62,37 @@ export const getClaims = (net, address) => {
         value: c.value
       }
     })
+    log.info(`Retrieved Claims for ${address} from neonDB ${net}`)
     return new Claims(claimData)
   })
 }
 
+/**
+ * Get total spent and unspent claimable amounts for an address.
+ * @param {string} net - 'MainNet' or 'TestNet'.
+ * @param {string} address - Address to check.
+ * @return {Promise<Claim>} An object with available and unavailable GAS amounts.

```

Comment:
`Promise<Fixed8>`

should change the description to be like maximum amount of gas claimable after spending all NEO.